modules/learn/pages/services-and-indexes/services/analytics-service.adoc

```diff
@@ -112,9 +112,13 @@ This continuous data ingestion allows operational and analytic queries to run co
 
 Due to the large scale and duration of operations it is likely to perform, the Analytics Service should be run _alone_, on its node or nodes in the cluster, _with no other Couchbase Service running on the Analytics nodes_.
 
-For the practical steps required to initialize or join a cluster, and to deploy services, see
-xref:manage:manage-nodes/create-cluster.adoc[Create a Cluster].
-For information on how to run analytic queries, see the xref:analytics:introduction.adoc[Introduction] to Couchbase Analytics.
+For more information about:
+
+* Initializing a cluster, see xref:manage:manage-nodes/create-cluster.adoc[Create a Cluster].
+
```

Comment:
Should add info for joining and adding a new node to a cluster also -- especially since on line 113, it says that the Analytics Service should be run alone on its own node or nodes in the cluster --  so, it's likely that people will want to add a new node.

`* Joining or adding a new node to a cluster, see see xref:manage:manage-nodes/join-cluster-and-rebalance.adoc[Join a Cluster and Rebalance] and see xref:manage:manage-nodes/add-node-and-rebalance.adoc[Add a Node and Rebalance].`